[2024-06-24T11:40:42.861+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T11:40:42.885+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.inject_data_kharkiv scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-24T11:40:42.898+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.inject_data_kharkiv scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-24T11:40:42.899+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-24T11:40:42.918+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): processing_tasks.inject_data_kharkiv> on 2024-06-23 00:00:00+00:00
[2024-06-24T11:40:42.927+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1781) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T11:40:42.929+0000] {standard_task_runner.py:63} INFO - Started process 1792 to run task
[2024-06-24T11:40:42.930+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'processing_tasks.inject_data_kharkiv', 'scheduled__2024-06-23T00:00:00+00:00', '--job-id', '190', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp1v8me2lo']
[2024-06-24T11:40:42.934+0000] {standard_task_runner.py:91} INFO - Job 190: Subtask processing_tasks.inject_data_kharkiv
[2024-06-24T11:40:42.998+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_dag.processing_tasks.inject_data_kharkiv scheduled__2024-06-23T00:00:00+00:00 [running]> on host afc8096e8b86
[2024-06-24T11:40:43.182+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='processing_tasks.inject_data_kharkiv' AIRFLOW_CTX_EXECUTION_DATE='2024-06-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-23T00:00:00+00:00'
[2024-06-24T11:40:43.185+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T11:40:43.185+0000] {sql.py:276} INFO - Executing: 
            INSERT INTO public.measures (timestamp, temp, humidity, clouds, wind_speed, city)
            VALUES (to_timestamp(1719100800), 
                   290.82,
                   76,
                   9,
                   0.45,
                   'Kharkiv'
            );
[2024-06-24T11:40:43.196+0000] {base.py:84} INFO - Using connection ID 'ps_db_conn' for task execution.
[2024-06-24T11:40:43.208+0000] {base.py:84} INFO - Using connection ID 'ps_db_conn' for task execution.
[2024-06-24T11:40:43.213+0000] {sql.py:487} INFO - Running statement: 
            INSERT INTO public.measures (timestamp, temp, humidity, clouds, wind_speed, city)
            VALUES (to_timestamp(1719100800), 
                   290.82,
                   76,
                   9,
                   0.45,
                   'Kharkiv'
            );, parameters: None
[2024-06-24T11:40:43.214+0000] {sql.py:496} INFO - Rows affected: 1
[2024-06-24T11:40:43.217+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T11:40:43.231+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=processing_tasks.inject_data_kharkiv, run_id=scheduled__2024-06-23T00:00:00+00:00, execution_date=20240623T000000, start_date=20240624T114042, end_date=20240624T114043
[2024-06-24T11:40:43.264+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T11:40:43.290+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24T11:40:43.293+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
